osc-server/src/main/java/org/osc/core/broker/service/validator/DeleteDistributedApplianceRequestValidator.java

```diff
@@ -62,4 +73,26 @@ public DistributedAppliance validateAndLoad(BaseDeleteRequest request) throws Ex
 
         return da;
     }
+
+    private void validateDAReferenceToSfcAndSecurityGroup(EntityManager em, DistributedAppliance da) throws Exception {
+		for (VirtualSystem vs : da.getVirtualSystems()) {
+			if (!vs.getServiceFunctionChains().isEmpty()) {
+				// DE5296-->Look if these SFCs are binded to any of the SG of same DA
```

Comment:
remove the comments

Reply:
Thank you for your review comments . I removed it.